Modules/TemperatureSensor/Message.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Azure.Amqp.Framing;
+
+namespace sendInfo
+{
+    class Message
+    {
+        public class NewValue
+        {
+            public String[] Headers;
+            public double[] Inputs;
+        }
+
+        public NewValue NewVal = new NewValue();

```

Comment:
find a more generic way to describe the data and feed the data in the visualizer, so next time we add more data to the flow, we won't have to refactor this